settings.gradle

```diff
@@ -1,5 +1,6 @@
 include ':auto-value-moshi'
 include ':auto-value-moshi-tests'
+include ':auto-value-moshi-annotations'
```

Comment:
Named the the artefact `auto-value-moshi-annotations` in case we will add new ones in the future.